core/actions/assertion.ts

```diff
@@ -89,7 +89,7 @@ export class Assertion extends ActionBuilder<dataform.Assertion> {
     }
 
     if (!config.name) {
-      config.name = Path.fileName(config.filename);
+      config.name = Path.basename(config.filename);
```

Comment:
i still think this capitalized import name is super weird (`Path` as opposed to `path` which would be more standard)

Reply:
It should probably be called `pathUtils` instead. It's capitalized for importing our path.ts file, but lowercase for our node path imports https://github.com/dataform-co/dataform/blob/b5368e04a047c6f2c958b6dd14d9bd5f872e4e0f/core/main_test.ts#L5.

Will update in a subsequent CL.

Reply:
IMO it's fine as `path` everywhere, but up to you